content/3_Silver/Binary_Search.problems.json

```diff
@@ -149,12 +162,24 @@
       "url": "https://cses.fi/problemset/task/1085",
       "source": "CSES",
       "difficulty": "Easy",
-      "isStarred": false,
+      "isStarred": true,
       "tags": ["Binary Search"],
       "solutionMetadata": {
         "kind": "internal"
       }
     },
+    {
+      "uniqueId": "ceoi-12-Job",
+      "name": "Job Scheduling",
```

Comment:
unique ID should be "ceoi-12-JobScheduling"

(by convention the entire name is camelcased)